html/login.html

```diff
@@ -3,7 +3,7 @@
             <link rel="stylesheet" href="../css/login.css">
         <title id="title">Login Page</title>
     </head>
-    <body style="background-image: url('../img/starfield.jpg')">
+    <body style="background-image: url('/Users/aaronsheoran/Desktop/Programing/starwars-search/images/backdrop.jpg')">
```

Comment:
shouldn't need the full path, should just be able to do:
```suggestion
    <body style="background-image: url('../images/backdrop.jpg')">
```